cf-applist.sh

```diff
@@ -267,6 +280,42 @@ json_apps=$(echo "$json_stacks"$'\n'"$json_spaces"$'\n'"$json_apps" | \
                      .extra.space = $spaces[.entity.space_guid].entity.name ) |
          .apps | {apps:.}')
 
+# Add extra data to json_routes
+json_routes=$(echo "$json_domains"$'\n'"$json_routes" | \
+     jq -s 'add' | \
+     jq '.domains as $domains |
+         .routes[] |= (.extra.domain = $domains[.entity.domain_guid].entity.name) |
+         .routes[] |= (.extra.route =
+           ( if .entity.host == "" then "" else .entity.host + "." end ) +
+           .extra.domain +
+           .entity.path
+         ) |
+         .routes | {routes:.}')
+
+# Join apps to routes
+echo "$json_domains" >/tmp/json_domains.txt
+echo "$json_routes" >/tmp/json_routes.txt
+echo "$json_route_mappings" >/tmp/json_route_mappings.txt
+echo "$json_apps" >/tmp/json_apps.txt
+
+json_apps=$(echo "$json_routes"$'\n'"$json_route_mappings"$'\n'"$json_apps" | \
+     jq -s 'add' | \
+     jq '.routes as $routes |
+         .route_mappings as $route_mappings |
+         .apps[] |= (
+           .metadata.guid as $app_guid |
+           .extra.froutes = [(
```

Comment:
ugly name, but I couldn't come up with something better: ideally this would be just `routes` and the `join` (314) would be done only when formatting the table